master/internal/core.go

```diff
@@ -305,6 +315,16 @@ func (m *Master) fetchAggregatedResourceAllocation(
 	}
 }
 
+// @Summary Get a detailed view of resource allocation during the given time period (CSV).
+// @Tags Cluster
+// @ID get-aggregated-resource-allocation-csv
+// @Produce  text/csv
+// @Param   start_date query string true "Start time to get allocations for"
+// @Param   end_date query string true "End time to get allocations for"
+// @Param   period query string true "Period to aggregate over"
```

Comment:
```suggestion
// @Param   start_date query string true "Start time to get allocations for (YYYY-MM-DD format for daily, YYYY-MM format for monthly)"
// @Param   end_date query string true "End time to get allocations for (YYYY-MM-DD format for daily, YYYY-MM format for monthly)"
// @Param   period query string true "Period to aggregate over (RESOURCE_ALLOCATION_AGGREGATION_PERIOD_DAILY or RESOURCE_ALLOCATION_AGGREGATION_PERIOD_MONTHLY)"
```
And maybe put `period` first to introduce the daily/monthly choice?